docs/app-check/usage/usage.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
This should be located at `docs/app-check/usage.md` instead of `docs/app-check/usage/usage.md` or alternatively it could be `docs/app-check/usage/index.md` - otherwise it 404s: https://react-native-firebase-git-mikehardy-appcheck-invertase.vercel.app/app-check/usage

Reply:
Other docs are in the format `docs/<module>/usage/index.md` - so this makes sense to keep it the same